meta-mbl-bsp-common/classes/mbl-kernel-config.bbclass

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+# This class adds a task to merge kernel config fragments into the main kernel config.
+# Recipes which add kernel config fragments should inherit this class.
+
+ARCH ?= "arm"
```

Comment:
Why we need this?

Reply:
We don't, I forgot to delete it after testing something else. Will remove.